lib/core/src/lib/form/components/form-custom-button.directive.ts

```diff
@@ -17,7 +17,8 @@
 
 import { Directive } from '@angular/core';
 
-/**
- * Directive selectors without adf- prefix will be deprecated on 3.0.0
- */
-@Directive({ selector: '[adf-form-custom-button], [form-custom-button]' }) export class StartFormCustomButtonDirective {}
```

Comment:
Looks like the old selector is still used in `start-form.component` @popovicsandras @eromano can it be safely removed?

Reply:
I missed that place, but can remove it from there. For external users it should be ok, because it is deprecated since 3.0 and now we have 6.7